scripts/hotWalletCli/wallet.ts

```diff
@@ -0,0 +1,336 @@
+        }
+
+        process.exit(1)
+      }
+    }
+
+    if (await isFunded()) return
+
+    const unsignedTx = await this.buildFundingTransaction(totalAmount, epoch.number)
+    const unsignedTxFile = path.join(RFOX_DIR, `unsignedTx_epoch-${epoch.number}.json`)
+
+    write(unsignedTxFile, JSON.stringify(unsignedTx, null, 2))
+    success(`Unsigned funding transaction created (${unsignedTxFile})`)
+
+    info(
+      `Follow the steps for signing and broadcasting the funding transaction as detailed here: https://github.com/shapeshift/rFOX/blob/main/scripts/hotWalletCli/MultiSig.md`,
+    )
+
+    const spinner = ora('Waiting for hot wallet to be funded...').start()
+
+    await (async () => {
+      return new Promise<void>(resolve => {
+        const interval = setInterval(() => {
+          isFunded(interval, spinner, resolve)
+        }, 30_000)
+      })
+    })()
+  }
+
+  private async signTransactions(epoch: Epoch): Promise<TxsByStakingAddress> {
+    const txsFile = path.join(RFOX_DIR, `txs_epoch-${epoch.number}.json`)
+    const txs = read(txsFile)
+
+    const totalTxs = Object.values(epoch.distributionsByStakingAddress).length
+    const spinner = ora(`Signing ${totalTxs} transactions...`).start()
+
+    const txsByStakingAddress = await (async () => {
+      if (txs) return JSON.parse(txs) as TxsByStakingAddress
+
+      const { address } = await this.getAddress()
+
+      const account = await (async () => {
+        try {
+          const { data } = await axios.get<{ account: { account_number: string; sequence: string } }>(
+            `${THORNODE_URL}/lcd/cosmos/auth/v1beta1/accounts/${address}`,
+          )
+          return data.account
+        } catch (err) {
+          spinner.fail()
+
+          if (isAxiosError(err)) {
+            error(
+              `Failed to get account details: ${err.request?.data?.message || err.response?.data?.message || err.message}, exiting.`,
+            )
+          } else {
+            error('Failed to get account details, exiting.')
+          }
+
+          process.exit(1)
+        }
+      })()
+
+      let i = 0
+      const txsByStakingAddress: TxsByStakingAddress = {}
+      try {
+        for await (const [stakingAddress, distribution] of Object.entries(epoch.distributionsByStakingAddress)) {
+          const unsignedTx = {
+            account_number: account.account_number,
+            addressNList,
+            chain_id: 'thorchain-mainnet-v1',
+            sequence: String(Number(account.sequence) + i),
+            tx: {
+              msg: [
+                {
+                  type: 'thorchain/MsgSend',
+                  value: {
+                    amount: [{ amount: distribution.amount, denom: 'rune' }],
+                    from_address: address,
+                    to_address: distribution.rewardAddress,
+                  },
+                },
+              ],
+              fee: {
+                amount: [],
+                gas: '0',
```

Comment:
confirming this is expected for TC fees

Reply:
This allows node to auto calc: 
```
curl "https://daemon.thorchain.shapeshift.com/lcd/cosmos/tx/v1beta1/txs/8253372B23093338EE65683E8E7F5D2F9D2192E3B782E972BFF8222ADFD3B80B"
```

note no amount or gas limit specified ^

https://viewblock.io/thorchain/tx/8253372B23093338EE65683E8E7F5D2F9D2192E3B782E972BFF8222ADFD3B80B
https://api.thorchain.shapeshift.com/api/v1/tx/8253372B23093338EE65683E8E7F5D2F9D2192E3B782E972BFF8222ADFD3B80B